go/vt/wrangler/materializer.go

```diff
@@ -578,16 +579,33 @@ func (wr *Wrangler) prepareCreateLookup(ctx context.Context, keyspace string, sp
 	}
 
 	// Validate input table
-	if len(specs.Tables) != 1 {
-		return nil, nil, nil, fmt.Errorf("exactly one table must be specified in the specs: %v", specs.Tables)
+	if len(specs.Tables) < 1 || len(specs.Tables) > 2 {
+		return nil, nil, nil, fmt.Errorf("one or two tables must be specified in the specs: %v", specs.Tables)
 	}
-	// Loop executes once.
+
+	// Loop executes once or twice
 	for k, ti := range specs.Tables {
 		if len(ti.ColumnVindexes) != 1 {
 			return nil, nil, nil, fmt.Errorf("exactly one ColumnVindex must be specified for the table: %v", specs.Tables)
```

Comment:
We should now instead specify the actual table name here rather than the spec.

Reply:
You mean in the error message?